cdap-docs/_common/common_conf.py

```diff
@@ -36,15 +36,24 @@
 
 def get_sdk_version():
     # Sets the Build Version
-    grep_version_cmd = "grep '<version>' ../../../pom.xml | awk 'NR==1;START{print $1}'"
     version = None
     short_version = None
     full_version = None
     try:
-        full_version = subprocess.check_output(grep_version_cmd, shell=True).strip().replace("<version>", "").replace("</version>", "")
+# Python 2.7 commands
+#         grep_version_cmd = "grep '<version>' ../../../pom.xml | awk 'NR==1;START{print $1}'"
+#         print "grep_version_cmd: %s" % grep_version_cmd
+#         full_version_temp = subprocess.check_output(grep_version_cmd, shell=True)
+# Python 2.6 commands
+        p1 = subprocess.Popen(["grep" , "<version>", "../../../pom.xml" ], stdout=subprocess.PIPE)
```

Comment:
When would you see that variable being set, and by which process? It's probably a good idea to change it, as this is fragile, as you've spotted. The script is a common script that gets copied and included; as I write this, I'm thinking that the including script (which is the conf.py file in each of the manuals) would then set the variable. In practise, I would set the variable by default to this, and a conf.py could over-ride it if necessary. As they are all at the same level right now, none would need to, but I could see a circumstance where it might be required.